rholang/src/main/scala/rholang/rosette/Compiler.scala

```diff
@@ -78,7 +78,22 @@ object Rholang2RosetteCompiler extends RholangASTToTerm
   }
 
   def main(args: Array[String]): Unit = {
-    val result = compile(args(0))
-    println(serialize(result))
+    args.toList match {
```

Comment:
Why do you have to convert to List here? Does array not support the match/case syntax? Quick google search suggests it does.

Reply:
No good reason. Indeed, I see `case Array(_, "", _*)` in https://stackoverflow.com/questions/6647166/how-do-i-pattern-match-arrays-in-scala

My first attempt to write an array pattern didn't compile and using `toList` seemed quicker than looking it up.

I can fix this.